frontend/src/AlertsManagement/AlertInfo.js

```diff
@@ -0,0 +1,83 @@
+import React, { Fragment } from 'react';
+import { Line } from 'react-chartjs-2';
+import { Link } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+
+const useStyles = makeStyles({
+  divider: {
+    marginTop: '20px',
+    marginBottom: '20px',
+  },
+});
+
+export default function AlertInfo(props) {
+  const classes = useStyles();
+  const alert = props.alert;
+
+  return (
+    <div className="alert-info">
+      <Divider variant="middle"/>
+      <Link to="/alerts">
+        <IconButton size="small" id="back-button">
+          <ArrowBackIcon />
+        </IconButton>
+      </Link>
+      <Typography variant="h5" align="center">
+        Alert on {alert.timestampDate}
+      </Typography>
+      <Typography variant="h6" align="center">
+          Status: {alert.status}
+        </Typography>
+      <List className="anomalies-list">
+        {alert.anomalies.map((anomaly, index) => {
+          const chartData = {
+            labels: [ ...anomaly.dataPoints.keys() ],
+            datasets: [
+              {
+                label: anomaly.metricName + ' for ' + anomaly.dimensionName,
+                fill: false,
+                lineTension: 0.1,
+                backgroundColor: 'rgba(75,192,192,0.4)',
```

Comment:
I'm not sure, I don't think so though because I have to generate a graph for each anomaly so the information (anomaly.metricName, anomaly.dataPoints, etc.) has to be inside the render...

Reply:
sounds good